cmd/exporter/exporter.go

```diff
@@ -74,6 +74,7 @@ func providerFlags(fs *flag.FlagSet, cfg *config.Config) {
 	flag.StringVar(&cfg.Providers.AWS.Region, "aws.region", "", "AWS region")
 	// TODO - PUT PROJECT-ID UNDER GCP
 	flag.StringVar(&cfg.ProjectID, "project-id", "ops-tools-1203", "Project ID to target.")
+	flag.StringVar(&cfg.Providers.Azure.SubscriptionId, "azure.subid", "", "Azure subscription ID to pull data from.")
```

Comment:
Any thoughts on just having this written out fully as `azure.subscription-id`? It seems awkward having the config spelt out explicity but not the cli argument. Eventually would could expose this as an environment variable as well to mitigate the need for the flag

Reply:
done